pages/billy-wilder-theater/index.vue

```diff
@@ -77,15 +108,343 @@ useHead({
 <template>
   <main
     id="main"
-    class="page page-billy-wilder-theater"
-    style="padding: 25px 100px;"
+    class="page page-detail page-detail--paleblue page-billy-wilder-theater"
   >
+    <div class="one-column">
+      <ResponsiveImage
+        v-if="parsedImage.length === 1"
+        :media="parsedImage[0]"
+        :aspect-ratio="43.103"
+        data-test="hero-image"
+      />
+    </div>
     <SectionWrapper>
-      <h2>Plan Your Visit</h2>
-      <pre style="text-wrap: auto;">{{ page }}</pre>
-      <divider-general />
+      <SectionHeader
+        :level="1"
+        class="page-heading"
+        data-test="page-heading"
+      >
+        {{ page.title }}
+      </SectionHeader>
+
+      <DividerWayFinder />
+
+      <SectionHeader
+        :level="2"
+        class="section-heading"
+      >
+        {{ parsedAdmissions.title }}
+      </SectionHeader>
+
+      <BlockInfo color-scheme="paleblue">
+        <template #block-info-top>
+          <RichText
+            :rich-text-content="parsedAdmissions.description"
+            data-test="admissions-intro"
+          />
+        </template>
+      </BlockInfo>
+
+      <div class="block-info-wrapper--flex admissions">
+        <BlockInfo
+          v-for="block in parsedAdmissions.blocks"
+          :key="block.title"
+        >
+          <template #block-info-top>
+            <SectionHeader
+              :level="3"
+              class="section-subtitle"
+            >
+              {{ block.title }}
+            </SectionHeader>
+          </template>
+          <template #block-info-mid>
+            <RichText
+              :rich-text-content="block.text"
+              data-test="admissions-info"
+            />
+          </template>
+        </BlockInfo>
+      </div>
+
+      <DividerWayFinder />
+
+      <SectionHeader
+        :level="2"
+        class="section-heading"
+      >
+        {{ parsedParking.title }}
+      </SectionHeader>
+
+      <div class="block-info-wrapper--flex directions">
+        <BlockInfo
+          color-scheme="paleblue"
+          class="address"
+          data-test="theater-address"
+        >
+          <template #block-info-top>
+            <p class="map-title">
+              {{ parsedParking.address.locationName }}
```

Comment:
{{ parsedParking?.address?.locationName }}

Reply:
I am thinking more like an editor. If I am adding Billy Wilder Theater data and I have preview open on the side, how would preview react to incomplete data?